<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1342</width>
    <height>782</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>257</width>
    <height>92</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1342</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File </string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionRestore"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="separator"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="separator"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionFilter"/>
    <addaction name="separator"/>
    <addaction name="actionEditor"/>
    <addaction name="separator"/>
    <addaction name="actionScript"/>
    <addaction name="separator"/>
    <addaction name="actionHook"/>
    <addaction name="separator"/>
    <addaction name="actionComand_Line"/>
    <addaction name="separator"/>
    <addaction name="actionHistorical_Copy"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionUser_Manual"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuEdit"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionFilter"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <widget class="QDockWidget" name="captureWindow">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_53">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Capture File</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="captureWindow_captureFile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>361</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="captureWindow_list">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>361</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Capture file in a list</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="FormatterWindow">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_60">
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Formatter List</string>
     </property>
    </widget>
    <widget class="QListWidget" name="FormatterWindow_formatterList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>371</width>
       <height>171</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Formatter1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Formatter2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Formatter3</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="FormatterWindow_applyFormatter">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>200</y>
       <width>151</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply formatter</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Rule for</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="FormatterWindow_ProtocolName"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Based on</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="FormatterWindow_filterName"/>
      </item>
     </layout>
    </widget>
    <widget class="QListWidget" name="FormatterWindow_ruleList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>371</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="FormatterWindow_actionList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>371</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>560</y>
       <width>371</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="FormatterWindow_createRule">
        <property name="text">
         <string>Create Rule</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="FormatterWindow_deleteRule">
        <property name="text">
         <string>Delete Rule</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Construction of New Rule with</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_4"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>1210</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Mode of Operation:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>781</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="main_window_filter">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Fiter</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Icons/myres.qrc">
     <normaloff>:/Icons/open.png</normaloff>:/Icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Icons/myres.qrc">
     <normaloff>:/Icons/save.png</normaloff>:/Icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="icon">
    <iconset resource="Icons/myres.qrc">
     <normaloff>:/Icons/restore.png</normaloff>:/Icons/restore.png</iconset>
   </property>
   <property name="text">
    <string>Restore</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="Icons/myres.qrc">
     <normaloff>:/Icons/close.png</normaloff>:/Icons/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Icons/myres.qrc">
     <normaloff>:/Icons/undo.png</normaloff>:/Icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Icons/myres.qrc">
     <normaloff>:/Icons/redo.png</normaloff>:/Icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="Icons/myres.qrc">
     <normaloff>:/Icons/copy.png</normaloff>:/Icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="Icons/myres.qrc">
     <normaloff>:/Icons/cut.png</normaloff>:/Icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="Icons/myres.qrc">
     <normaloff>:/Icons/paste.png</normaloff>:/Icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionFilter">
   <property name="icon">
    <iconset resource="Icons/myres.qrc">
     <normaloff>:/Icons/filter.png</normaloff>:/Icons/filter.png</iconset>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
  </action>
  <action name="actionEditor">
   <property name="icon">
    <iconset resource="Icons/myres.qrc">
     <normaloff>:/Icons/editor.png</normaloff>:/Icons/editor.png</iconset>
   </property>
   <property name="text">
    <string>Editor</string>
   </property>
  </action>
  <action name="actionScript">
   <property name="icon">
    <iconset resource="Icons/myres.qrc">
     <normaloff>:/Icons/script.png</normaloff>:/Icons/script.png</iconset>
   </property>
   <property name="text">
    <string>Script</string>
   </property>
  </action>
  <action name="actionHook">
   <property name="text">
    <string>Hook</string>
   </property>
  </action>
  <action name="actionComand_Line">
   <property name="icon">
    <iconset resource="Icons/myres.qrc">
     <normaloff>:/Icons/command.png</normaloff>:/Icons/command.png</iconset>
   </property>
   <property name="text">
    <string>Comand Line</string>
   </property>
  </action>
  <action name="actionHistorical_Copy">
   <property name="icon">
    <iconset resource="Icons/myres.qrc">
     <normaloff>:/Icons/historical.png</normaloff>:/Icons/historical.png</iconset>
   </property>
   <property name="text">
    <string>Historical Copy</string>
   </property>
  </action>
  <action name="actionUser_Manual">
   <property name="text">
    <string>User Manual</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Icons/myres.qrc"/>
 </resources>
 <connections/>
</ui>
